charts/thoras/values.yaml

```diff
@@ -53,4 +53,15 @@ thorasDashboard:
     memory: 100Mi
   port: 80
 
+thorasMonitor:
+  enabled: false
+  monitorCadence: "15"
```

Comment:
Can we somehow expose units either through key or value? I think users won't know what to do without at least a hint. `monitorCadenceSeconds` seems like a pretty easy way to achieve this with no app changes